examples/rnn/LSTM.hs

```diff
@@ -60,24 +60,16 @@ instance RecurrentCell LSTMCell where
       cNew = newCellState cell input hidden
 
 
+
 instance Randomizable LSTMSpec LSTMCell where
   sample LSTMSpec{..} = do
-      ig_ih <- makeIndependent =<< randn' [inf, hf]
-      ig_hh <- makeIndependent =<< randn' [hf, hf]
-      ig_b <- makeIndependent =<< randn' [1, hf]
-      fg_ih <- makeIndependent =<< randn' [inf, hf]
-      fg_hh <- makeIndependent =<< randn' [hf, hf]
-      fg_b <- makeIndependent =<< randn' [1, hf]
-      og_ih <- makeIndependent =<< randn' [inf, hf]
-      og_hh <- makeIndependent =<< randn' [hf, hf]
-      og_b <- makeIndependent =<< randn' [1, hf]
-      hg_ih <- makeIndependent =<< randn' [inf, hf]
-      hg_hh <- makeIndependent =<< randn' [hf, hf]
-      hg_b <- makeIndependent =<< randn' [1, hf]
-      let ig = [ig_ih, ig_hh, ig_b]
-      let fg = [fg_ih, fg_hh, fg_b]
-      let og = [og_ih, og_hh, og_b]
-      let hg = [hg_ih, hg_hh, hg_b]
+      w_ih <- makeIndependent =<< randn' [inf, hf]
+      w_hh <- makeIndependent =<< randn' [hf, hf]
+      b <- makeIndependent =<< randn' [1, hf]
+      let ig = [w_ih, w_hh, b]
+      let fg = [w_ih, w_hh, b]
+      let og = [w_ih, w_hh, b]
+      let hg = [w_ih, w_hh, b]
```

Comment:
These codes works at first step only.
In training loop, we call makeIndependent for each parameters.
https://github.com/hasktorch/hasktorch/blob/master/examples/elman/Main.hs#L48
So if you want to merge these variables, you should remove duplicated variable of data-LSTM or Paramertrized's one.

Reply:
@AdLucem if you can fix this and revert the seq2seq commit so that's in a different feature branch I can merge this PR.

Reply:
Fixed!